app/serializers/activitypub/image_serializer.rb

```diff
@@ -20,6 +20,8 @@ def media_type
     object.content_type
   end
 
+  delegate :name, to: :object
```

Comment:
At the moment, this just returns `"avatar"` or `"header"` as the name; I'd suggest handling this like `focal_point` and using `object.instance_respond_to?` and `object.instance_read` to check for and get the description value respectively.